evalai/utils/submissions.py

```diff
@@ -86,7 +88,9 @@ def display_my_submission_details(challenge_id, phase_id):
     except requests.exceptions.HTTPError as err:
         if (response.status_code in EVALAI_ERROR_CODES):
             validate_token(response.json())
-            echo(style("Error: {}".format(response.json()["error"]), fg="red", bold=True))
+            echo(style("\nError: {}".format(response.json()["error"]), fg="red", bold=True))
+            echo(style("\nUse `evalai challenges` to fetch the active challenges.", fg="red", bold=True))
+            echo(style("\nUse `evalai challenge CHALLENGE phases` to fetch the active phases.\n", fg="red", bold=True))
```

Comment:
Any reason this line repeated so many times? Is there no better way to do this?

Asking so I can understand your thought process and recommend approaches. :)

Reply:
Now that I think about it, 
```
echo(style("\nError: {}"
"Use `evalai challenges` to fetch the active challenges."
"Use `evalai challenge CHALLENGE phases` to fetch the active"
"phases.\n".format(response.json()["error"]),
fg="red", bold=True))
```

Looks much better. 

What do you think @varunagrawal @RishabhJain2018 ?